lib/govuk_publishing_components/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukPublishingComponents
-  VERSION = '9.16.1'.freeze
+  VERSION = '9.17'.freeze
```

Comment:
For the future: this should have been `9.17.0`, as that's [the semver way](https://semver.org/).

Reply:
Yep, forgot the patch. My bad.